### PR TITLE
"Chi2 / ndof" plotting in makePlot.py

### PR DESCRIPTION
**Pull request by cesar-avalos3**
Add the chi2/ndof functionality lifted from “plotFitResults.py”

### CODE REVIEW

File: Scripts/makePlots.py [hunk 1]
```diff
@@ -307,11 +236,19 @@ def addPlots(description, paramName, param, paramErr, ouFileName):
...
-    canvas = r.TCanvas()
-    ratioPlots(description, paramName, param1, param2, outFileName)
+    paramName = "chi2/ndof"
+    chi2 = fitResult.getFitParam("chi2")
+    ndof = fitResult.getFitParam("ndof")
+    for a in chi2:
+        for b in chi2[a]:
+            num = chi2[a][b]
+            denom = ndof[a][b]
+            param[a][b] = num/denom
```
Comment (gkrintir) [inline Scripts/makePlots.py]: Could you please add an exception here ?

**Comment by gkrintir** [comment]
Hi @cesar-avalos3 Just add an exception on the division with ndof.

Since it is of the same nature, could you please add the very same exception in the plotFitResults.py macro ? Thanks ! ;D

**Comment by cesar-avalos3** [comment]
Hey gkrintir.
Done and done.

File: Scripts/makePlots.py [hunk 2]
```diff
@@ -223,15 +227,37 @@ def addPlots(description, paramName, param, paramErr, ouFileName):
...
+    ndof  = fitResult.getFitParam("ndof")
+    for a in chi2:
+        for b in chi2[a]:
+            num = chi2[a][b]
+            denom = ndof[a][b]
+            try:
+                param[a][b] = num/denom
+            except ZeroDivisionError:
+                # dummy zero value
+                param[a][b] = 0
```
Comment (gkrintir) [inline Scripts/makePlots.py]: dummy zero value -->  dummy value (-999)
param[a][b] = 0 --> param[a][b] = -999

File: Scripts/plotFitResults.py [hunk 3]
```diff
@@ -228,25 +228,30 @@ def addPlots(description, paramName, param, paramErr, ouFileName):
...
     for a in chi2:
         for b in chi2[a]:
             num = chi2[a][b]
             denom = ndof[a][b]
-            param[a][b] = num/denom
+            try:
+                param[a][b] = num/denom
+            except ZeroDivisionError:
+                # dummy zero value
+                param[a][b] = 0
```
Comment (gkrintir) [inline Scripts/plotFitResults.py]: dummy zero value -->  dummy value (-999)
param[a][b] = 0 --> param[a][b] = -999

**Comment by gkrintir** [comment]
Hi @cesar-avalos3  thanks a lot! A last detail, please use as dummy value -999, since 0 could have a potential real meaning ;D

**Comment by gkrintir** [comment]
Thanks again ;D

**Comment by cesar-avalos3** [comment]
This is true, fixed it. 
Appreciated

**Pull request merged**